lib/mayaUsd/python/wrapUtil.cpp

```diff
@@ -50,5 +50,6 @@ void wrapUtil()
     scope s(class_<UsdMayaUtilScope>("Util"));
 
     def("IsAuthored", UsdMayaUtil::IsAuthored);
+    def("prettifyName", UsdMayaUtil::prettifyName);
```

Comment:
(minor) - why is IsAuthored capitalized while prettifyName is lower case?

Reply:
The coding standards requires CamelCase, I will update the code.

Reply:
Yes. I rechecked and it is indeed lowerCamelCase.